src/__tests__/scenes/__snapshots__/CryptoExchangeQuoteScene.test.tsx.snap

```diff
@@ -1393,5 +1393,17 @@ exports[`CryptoExchangeQuoteScreenComponent should render with loading props 1`]
       </View>
     </RCTScrollView>
   </View>
+  <View
```

Comment:
Don't these test changes go with the previous commit? This one didn't change any JSX.

Reply:
I went back and updated each commit with `yarn test -u`